app/components/app-chrome.hbs

```diff
@@ -14,7 +14,7 @@
         {{#if @editorDocument.updatedOn}}
         <li class="au-c-list-horizontal__item">
           <span class="au-c-app-chrome__status">
-            Bewaard op {{moment-calendar @editorDocument.updatedOn}}
+            Bewaard op {{human-friendly-date @editorDocument.updatedOn}}
```

Comment:
What does this look like in practice?  Doesn't this result in things like `Gisteren` or `2 dagen geleden`?

If so, doesn't `Bewaard op gisteren` sound wrong? Maybe I'm misunderstanding the output :smile:

Reply:
I agree, but this is also the case with moment-calender and for now I just want to match old behaviour

Reply:
I assumed it was a mistake in the previous implementation but fair enough :smile: